tests/Microsoft.Identity.Web.Test/WebApiExtensionsTests.cs

```diff
@@ -65,7 +72,11 @@ public void AddProtectedWebApi_WithConfigName()
             provider.GetRequiredService<IOptionsFactory<MicrosoftIdentityOptions>>().Create(string.Empty);
             config.Received(3).GetSection(_configSectionName);
 
-            AddProtectedWebApi_TestCommon(services, provider);
+            // Ideally we'd want to configure the GetValue for "TokenDecryptionCertificates" but that's not possible!
```

Comment:
@pmaytak : how can we substitute so that the value of TokenDecryptionCertificates is right? 
I'm even questioning if we want to substitute the config. Why not directly use the configuration methods?

Reply:
Well we're mocking config to test that the config was bound (for the overloads with config name). We probably don't need to test that because configuration values are inherently tested in the methods themselves when they are used. So we can just provide test config data without mocking. To get the value for any config property, could just do something like this:
`provider.GetRequiredService<IOptionsFactory<MicrosoftIdentityOptions>>().Create(string.Empty);`